wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -99,10 +98,15 @@ const Lobby = ({location}) => {
                     :
                     <h2>Waiting for Host to start!</h2> 
                 }
-                { goSwipe ? <Redirect to={{
-                    pathname: '/swiping',
-                    search: `?room=${roomId}`,
-                    state: {room: roomId}}}/> : null }
+                { goSwipe && topTenMovies!=null ? <Redirect to={{ 
+                                pathname: '/swiping',
+                                search:'?room=${roomId}',
```

Comment:
I think we should replace the single quotes (' ') here with backticks (` `) so that roomId will be populated correctly in the URL parameters.

Reply:
Good catch!